docs/concepts/models.md

```diff
@@ -524,182 +500,126 @@ except ValidationError as e:
-* For models with private attributes, the `__pydantic_private__` dict will be initialized the same as it would be when
-  calling `__init__`.
-* When constructing an instance using [`model_construct()`][pydantic.main.BaseModel.model_construct], no `__init__` method from the model or any of its parent
-  classes will be called, even when a custom `__init__` method is defined.
-
-!!! note "On `extra` behavior with `model_construct`"
-    * For models with `model_config['extra'] == 'allow'`, data not corresponding to fields will be correctly stored in
-    the `__pydantic_extra__` dict and saved to the model's `__dict__`.
-    * For models with `model_config['extra'] == 'ignore'`, data not corresponding to fields will be ignored - that is,
+* For models with private attributes, the `__pydantic_private__` dictionary will be populated the same as it would be when
+  creating the model with validation.
+* No `__init__` method from the model or any of its parent classes will be called, even when a custom `__init__` method is defined.
+
+!!! note "On [extra fields](#extra-fields) behavior with [`model_construct()`][pydantic.main.BaseModel.model_construct]"
+    * For models with [`extra`][pydantic.ConfigDict.extra] set to `'allow'`, data not corresponding to fields will be correctly stored in
+    the `__pydantic_extra__` dictionary and saved to the model's `__dict__` attribute.
+    * For models with [`extra`][pydantic.ConfigDict.extra] set to `'ignore'`, data not corresponding to fields will be ignored — that is,
     not stored in `__pydantic_extra__` or `__dict__` on the instance.
-    * Unlike a call to `__init__`, a call to `model_construct` with `model_config['extra'] == 'forbid'` doesn't raise an
-    error in the presence of data not corresponding to fields. Rather, said input data is simply ignored.
+    * Unlike when instiating the model with validation, a call to [`model_construct()`][pydantic.main.BaseModel.model_construct] with [`extra`][pydantic.ConfigDict.extra] set to `'forbid'` doesn't raise an error in the presence of data not corresponding to fields. Rather, said input data is simply ignored.
 
 ## Generic models
 
 Pydantic supports the creation of generic models to make it easier to reuse a common model structure.
 
-In order to declare a generic model, you perform the following steps:
+In order to declare a generic model, you should follow the following steps:
 
-1. Declare one or more `typing.TypeVar` instances to use to parameterize your model.
-2. Declare a pydantic model that inherits from `pydantic.BaseModel` and `typing.Generic`,
-  where you pass the `TypeVar` instances as parameters to `typing.Generic`.
-3. Use the `TypeVar` instances as annotations where you will want to replace them with other types or
-  pydantic models.
+1. Declare one or more [type variables][typing.TypeVar] to use to parameterize your model.
+2. Declare a pydantic model that inherits from [`BaseModel`][pydantic.BaseModel] and [`typing.Generic`][],
```

Comment:
Maybe specify "in that order" unless we specify elsewhere, I remember this being a point of confusion for folks